Add protocols as top-level directory with third_party protocols (#13)
robocin/ssl-core: protocols/README.md

# [`ssl-core`](https://github.com/robocin/ssl-core) Protocols

The protocols directory is a central hub dedicated to storing and managing protocols that are intended to be shared across multiple projects. It serves as a reliable source of the latest versions of the protocols definitions, providing an organized directory structure that categorizes and documents each protobuf file, making it simple for developers to navigate and understand the available data structures.

By centralizing protocols in a dedicated folder, it promotes code reuse, modularity, and efficient development practices. Any updates or modifications to the shared protocols can be made in a controlled manner, ensuring compatibility across multiple repositories that depend on these protocols.

## About

The protocols contained in this repository are Protocol Buffers (protobuf).

### Protobufs

> [Protobufs](https://protobuf.dev) are a language-agnostic data serialization format developed by Google, which allow for efficient and structured communication between different software components.

Reviewer: ```suggestion
[Protobufs](https://protobuf.dev) are a language-agnostic data serialization format developed by Google that allows  efficient and structured communication between different software components.
```
